app/assets/stylesheets/forms.scss

```diff
@@ -34,3 +34,8 @@ button {
   @extend .btn;
   @extend .btn-default;
 }
+
+#new_mainstream_browse_page_archival_form,
```

Comment:
@mobaig These two classes are auto-generated by Rails based on the views, so they always use underscores.